_component_design/text-inputs.md

```diff
@@ -13,185 +13,291 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-# Text Field
-
-Text fields are generally found in forms. The type of text field determines
-which characters are allowed within it. For example, a user shouldn't be able
-to enter a letter into a phone number field. When the user is entering a name
-or proper noun, the first letter should automatically be capitalized when
-typed.
-
-If the content is longer than the width of the text field, automatically scroll
-the content left as the user continues to type. Consider using a text area if
-this is expected to happen frequently in a specific text field.
-
-## Enabled
-
-<div class="ui form">
-  <div class="field required">
-    <label>Required field</label>
-    <input type="text" placeholder="This is placeholder text">
-  </div>
-  <div class="field">
-    <label>Field with subtext</label>
-    <input type="text" placeholder="This is placeholder text">
-    <div class="ui message subtext">
-    <p>This is some subtext.</p>
-    </div>
-  </div>
-  <div class="field">
-    <label>Prefix or suffix</label>
-    <div class="ui right labeled input">
-      <div class="ui label">$</div>
-      <input type="text" placeholder="Amount">
-      <div class="ui basic label">.00</div>
-    </div>
-  </div>
-  <div class="field">
-    <label>With icon left</label>
-    <div class="ui left icon input">
-      <input type="text" placeholder="Search">
-      <i class="search icon"></i>
-    </div>
-  </div>
-  <div class="field required">
-    <label>Password</label>
-    <input type="password" value="password">
-  </div>
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### When to use
+  * Use text fields to enable users to enter text, numbers, passwords, emails, logging in, copying API keys, and any other data pertinent to accomplishing their tasks.
+
+### Best practices
+  * Text fields should indicate state – whether enabled or disabled, empty or filled, valid or invalid – with clear label, input,  and assistive text.
```

Comment:
Editorial: Remove spaces around dash, and change to em dash:
```markdown
should indicate state&emdash;whether enabled or disabled, empty or filled, valid or invalid&emdash;with clear
```

Reply:
👍